packages/quantic/force-app/main/default/lwc/quanticAriaLive/quanticAriaLive.js

```diff
@@ -0,0 +1,32 @@
+import { LightningElement, track, api } from 'lwc';
+
+export default class QuanticAriaLive extends LightningElement {
```

Comment:
If we want this component to be visible in the reference docs its class needs proper comments.

Reply:
Interesting, do we want to?

Reply:
Anyway I created one